burn-import/src/burn/node/add.rs

```diff
@@ -0,0 +1,83 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, Type};
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Debug, Clone, new)]
+pub struct AddNode {
+    pub lhs: TensorType,
+    pub rhs: TensorType,
+    pub output: TensorType,
+}
```

Comment:
This will be very similar for all unary operations. I think it would be interesting to have a way to avoid code duplication with a trait that only defines the forward method. It may be done in another PR.

Reply:
Yes, that could be an idea. I agree